fileHeader_test.go

```diff
@@ -96,17 +96,81 @@ func TestFileHeader__ImmediateOrigin(t *testing.T) {
 	if v := header.ImmediateOriginField(); v != header.ImmediateOrigin {
 		t.Errorf("got %q", v)
 	}
+}
 
-	// make sure our trim works that we hook into FileHeader.Parse(..)
-	if v := trimImmediateOriginLeadingZero("0123456789"); v != "123456789" {
-		t.Errorf("got %q", v)
-	}
-	if v := trimImmediateOriginLeadingZero("0012345678"); v != "012345678" {
-		t.Errorf("got %q", v)
-	}
-	zeros := strings.Repeat("0", 10)
-	if v := trimImmediateOriginLeadingZero(zeros); v != zeros {
-		t.Errorf("got %q", v)
+func TestFileHeader__ImmediateDestination(t *testing.T) {
```

Comment:
These tests are almost the same as the test above for `ImmediateOrigin` but we're using table-driven tests here for clarity; let me know if I should spin up another issue to convert the tests to match.

Reply:
This nice cleanup, thanks!